misfitapp/models.py

```diff
@@ -168,7 +168,7 @@ class Device(MisfitModel):
 
     id = models.CharField(max_length=MAX_KEY_LEN, primary_key=True)
     user = models.ForeignKey(UserModel)
-    device_type = models.CharField(choices=DEVICE_TYPES, max_length=5)
+    device_type = models.CharField(choices=DEVICE_TYPES, max_length=64)
```

Comment:
@percyperez Can we get rid of the these `choices` as well? `DEVICE_TYPES` isn't an accurate list of possibilities, so it is confusing to keep it around.